zzupy/aio/app/eas/undergrad.py

```
@@ -0,0 +1,378 @@
+    def __init__(self, cas_client: ICASClient):
+
+        if not cas_client.logged_in:
+            raise NotLoggedInError("CASClient 必须已经登录")
+
+        def request_logger(request):
+            logger.trace(f">>> {request.method} {request.url}")
+            logger.trace(f">>> Headers: {dict(request.headers)}")
+            if request.content:
+                try:
+                    logger.trace(f">>> Body: {request.content.decode('utf-8')}")
+                except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)